homu/server.py

```diff
@@ -44,7 +44,8 @@ class G:
 g = G()
 
 ROLLUP_STR = {
-    -1: 'never',
+    -2: 'never',
+    -1: 'iffy',
     0: '',
     1: 'always',
```

Comment:
It might be nice to generate this from `WORDS_TO_ROLLUP`.

Reply:
I tried, I couldn't find a sensible place to do this that didn't cause cyclic imports. We can't do it in main.py